### PR TITLE
Extend documentation of directories setting

### PR DESCRIPTION
**Pull request by orontee**
Relates to https://github.com/mopidy/mopidy-local/issues/38

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -173,6 +173,110 @@ To delete your local images and clear your local library::
...
+
+Browsing URIs starting with ``local:artist`` returns references to albums with
+the given artist and tracks with that artist. Browsing URIs starting with
+``local:album`` returns references to the album tracks. Browsing URIs starting
+with ``local:track`` is not supported.
+
+Other URIs supported for browsing start with ``local:directory``. The returned
+references are specified through "query parameters":
+
+- ``local:directory``: References to top levels directories.
```
Comment (kingosticks) [inline README.rst:218]:
```suggestion
- ``local:directory``: Reference to top-level directories.
```

File: README.rst [hunk 2]
```diff
@@ -173,6 +173,110 @@ To delete your local images and clear your local library::
...
+- ``local:directory?genre=GENRE``: References to directories grouping tracks
+  whose genre is GENRE by album. Their URIs start with
+  ``local:directory?genre=GENRE&type=track&album=local:album:``.
+
+- ``local:directory?performer=URI``: References to directories grouping tracks
+  whose performer has given URI by album. Their URIs start with
+  ``local:directory?performer=URI&type=track&album=local:album:``.
+
+- ``local:directory?max-age=SECONDS``: References to directories grouping tracks
+  whose date is newer than SECONDS seconds by album. Their URIs start with
```
Comment (kingosticks) [inline README.rst:277]: I think it's actually be the date it was scanned, not the date of the track itself.

**Review by kingosticks — CHANGES_REQUESTED** [review]
This is useful, thank you. Saves everyone looking through the code and/or banging their head against a wall.

**Comment by orontee** [comment]
Thanks for your suggestions. I'll rework this at the end of the week, I don't have access to a computer for few days.

**Comment by orontee** [comment]
The proposed change has been rebased on `master`, you suggestions taken into account and I made some rewording. A wrong URI has been corrected too.

File: README.rst (hunk 2 above)
Comment (orontee) [reply README.rst]: You're right.

File: README.rst [hunk 3]
```diff
@@ -173,6 +173,122 @@ To delete your local images and clear your local library::
...
+  found among all tracks. Their URIs start with ``local::directory?genre=``.
+
+- ``local:directory?type=album``: References to all albums.
+
+- ``local:directory?type=album&PARAM=VALUE``: References to
+  directories grouping tracks matching the given criteria.  ``PARAM``
+  must be one of ``albumartist``, ``artist``, ``composer``, ``date``,
+  ``genre``, ``performer``, ``max-age``. The referenced directories
+  group the selected tracks by album; Their URIs start with
+  ``local::directory?PARAM=VALUE&type=track&album=local:album:``.
```
Comment (jodal) [inline README.rst:240]:
```suggestion
  ``local:directory?PARAM=VALUE&type=track&album=local:album:``.
```

File: README.rst [hunk 4]
```diff
@@ -173,6 +173,122 @@ To delete your local images and clear your local library::
...
+  ``albumartist``, ``artist``, ``composer``, ``date``, ``genre``,
+  ``performer``, and ``max-age``.
+
+- ``local:directory?type=date``: References to directories grouping tracks by
+  date and album. Dates are transformed according to the optional parameter
+  ``FORMAT`` which default to ``%Y-%m-%d``. The URIs of the references start
+  with ``local:directory?date=``.
+
+- ``local:directory?type=genre``: References to directories named after genres
+  found among all tracks. Their URIs start with ``local::directory?genre=``.
```
Comment (jodal) [inline README.rst:231]:
```suggestion
  found among all tracks. Their URIs start with ``local:directory?genre=``.
```

**Review by jodal — APPROVED** [review]
Two small errors I found.

Could you also add the `Library layout` header to the table of contents up top?

**Comment by jodal** [comment]
Thanks for taking the time and effort to document this properly!

**Pull request merged**